assets/js/components/HostDetails/HostDetails.test.jsx

```diff
@@ -66,4 +67,115 @@ describe('HostDetails component', () => {
       )
     ).toBeInTheDocument();
   });
+
+  it("should not display 'Clean up' button when host is not deregisterable", () => {
```

Comment:
double quotes? I guess the ci will complain about this hehe
I would remove the single quotes in any case from `Clean up` and put them in the test string